concordia/migrations/0026_creategroups.py

```diff
@@ -0,0 +1,28 @@
+from django.db import migrations
+from django.conf import settings
+
+
+def create_groups(apps, schema_editor):
+    Group = apps.get_model("auth", "Group")
+
+    cm_group_exists = False
+    newsletter_group_exists = False
+
+    try:
```

Comment:
I don't really care that much but this could be `Group.objects.get_or_create()` without the `try`/`except` block